*: add specific Helm version (#1820)
pingcap/docs: v2.1-legacy/op-guide/docker-compose.md

1. Install Helm.

>     [Helm](https://helm.sh) can be used as a template rendering engine. Currently we only support Helm version >= 2.9.0 and < 3.0.0. The installation steps are as follows:

Reviewer: ```suggestion
    [Helm](https://helm.sh) can be used as a template rendering engine. Make sure your Helm version >= 2.9.0 and < 3.0.0. The installation steps are as follows:
```
